packages/dts-plugin/src/core/lib/DTSManager.advance.spec.ts

```diff
@@ -70,6 +70,7 @@ describe('DTSManager advance usage', () => {
       remoteOptions.typesFolder,
     );
     await dtsManager.generateTypes();
+    console.log('generateTypes done');
```

Comment:
Does this need to be removed?

Reply:
no, this test occasionally goes wrong. I need to add this log for debugging.